src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9355,9 +9359,9 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
         }
     } else if (VRFCoolingMode || HRCoolingMode) {
         // IF the system is in cooling mode and/or the terminal unit requests cooling
-        if (NoCompOutput <= QZnReq) {
-            PartLoadRatio = 0.0;
-            return;
+        if (NoCompOutput <= QZnReq || (QZnReq >= -DataHVACGlobals::SmallLoad && NoCompOutput <= QZnReq && !HRCoolingMode)) {
```

Comment:
The `||` will never get used here, because `NoCompOutput <= QZnReq` is on both sides, if it's true the `||` will never be evaluated. If it's false, the `||` clause will be false. Are these extra conditions necessary? If so, perhaps you just need the `||` clause and not the first part?

Reply:
I will test this again. These changes were incrementally added to address the setpoint temperature overshoots and undershoots.